src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -57,11 +59,82 @@ public class DefaultControllerActionArgumentBinder : IControllerActionArgumentBi
                 }
             }
 
+            var controllerType = actionDescriptor.ControllerTypeInfo.AsType();
+            var propertyMetadata = new List<ModelMetadata>();
+            foreach (var property in actionDescriptor.CommonParameters)
+            {
+                var metadata = _modelMetadataProvider.GetMetadataForProperty(
+                    controllerType,
+                    propertyName: property.Name);
```

Comment:
This is going to cause a conflict with my changes, not sure what we want to do about it. MM is now immutable.